modules/apps/object/object-service/service.xml

```diff
@@ -602,6 +602,7 @@
 		<exception>ObjectValidationRuleEngine</exception>
 		<exception>ObjectValidationRuleName</exception>
 		<exception>ObjectValidationRuleScript</exception>
+		<exception>ObjectViewColumn</exception>
```

Comment:
@p-albuquerque Can you create exceptions for each one of the ObjectViewColumn properties? You can follow the same example as ObjectValidationRule exceptions. 
Check how they're thrown in the LocalService classes.

cc @mateussgn